Add a colon to better separate a heading and its corresponding text (#72927)
swiftlang/swift: docs/HowToGuides/FAQ.md

[GitHub - About Git rebase](https://docs.github.com/en/get-started/using-git/about-git-rebase) 
provides a comprehensive overview of `rebase`.

> **Warning:**

> We suggest considering to `rebase` only those commits that haven't been pushed to a public branch. 
> Rebasing existing commits would block merging because we don't allow force pushes to the repository.
> If you need to tidy up commits that have already been pushed, 

Reviewer: Side note: This part is misleading because rebasing commits pushed to a fork is OK.
```suggestion
We suggest considering to `rebase` only those commits that are not part of the base repository. 
Rebasing existing commits would block merging because we don't allow force pushes to the repository.
If you need to tidy up commits that have already been merged into the base repository, 
```
